rpcs3/Emu/Cell/Modules/sceNpTrophy.cpp

```diff
@@ -861,7 +861,7 @@ error_code sceNpTrophyUnlockTrophy(u32 context, u32 handle, s32 trophyId, vm::pt
 	const std::string& config_path = vfs::get("/dev_hdd0/home/" + Emu.GetUsr() + "/trophy/" + ctxt->trp_name + "/TROPCONF.SFM");
 	const u32 unlocked_platinum_id = ctxt->tropusr->GetUnlockedPlatinumID(trophyId, config_path);
 
-	if (unlocked_platinum_id != 0u + SCE_NP_TROPHY_INVALID_TROPHY_ID)
+	if (unlocked_platinum_id != umax)
```

Comment:
😡 at least add a comment with SCE_NP_TROPHY_INVALID_TROPHY_ID at both places

Reply:
Okay, I made const unsigned since it's only used here